### ISSUE TITLE
Cant we create a new repo with this module??

### ISSUE THREAD

**Issue by vikramalmosafer** [OpenRailAssociation/github-org-manager#51]
do we also able to create repositories if not exist with this module?

**Comment by mxmehl** [comment]
No, that is currently not possible. I'm not even sure it's wanted as at least for our use-case, we also move repos from other orgs to this GitHub org. If we accidentally ran the tool with team permissions to a currently non-existing repo, it may block migrations.

With the current behaviour, we would probably just get an error on the tool's side, but it wouldn't do anything it shouldn't do.

However, please feel free to explain your use-case, or contribute a PR.

**Comment by vikramalmosafer** [comment]
i was planning to use this tool for managing multiple team and their respective repos which also include repo permission management and creation if not exists.

**Comment by mxmehl** [comment]
OK, it might make sense, perhaps we could make this configurable.

I won't get to do this before my parental leave, so not before the end of the year. If you would like to submit a PR, I'd be happy to review.
